### ISSUE TITLE
Change actuator serial port specs to Arduino

### ISSUE THREAD

**Issue by sadmansk** [uwnrg/minotaur-cpp#18]
Specifications:
Baud rate has to be set to `9600`. And the data sent to the arduino should be a byte array of size 4, where:
```
byte[0]: 1 for up,    -1 for down, 0 for no movement on y-axis
byte[1]: 1 for right, -1 for left, 0 for no movement on x-axis
byte[2]: (in seconds) how long the command should be kept on
byte[3]: speed (unit yet to be determined)
```

**Comment by sadmansk** [comment]
Is this closed by #60?

**Comment by Mogball** [comment]
More or less, all the actuator stuff remains to be removed from the code
